proxy/http/HttpSM.h

```diff
@@ -519,6 +522,18 @@ class HttpSM : public Continuation, public PluginUserArgs<TS_USER_ARGS_TXN>
   /// Update the milestones to track time spent in the plugin API.
   void milestone_update_api_time();
 
+  sockaddr *
+  get_server_ip() const
+  {
+    return &t_state.current.server->dst_addr.sa;
+  };
+
+  int
+  get_request_method() const
```

Comment:
If this returns a WKS index, it should indicate that  in the method name, just like `method_get_wksidx`.